playbooks/engineer-workflow.md

```diff
@@ -141,13 +153,18 @@ Reviewers are asked to leave their review within one business day of being reque
 any reason, it's the reviewer's responsibility to communicate with the author of the PR to establish a different
 timeline or get someone else to review.
 
-**_Assign_** a single team member (they will often also be one of your reviewers) to the PR. This is the person who
-is responsible for moving the PR forward.
+**_Assign_** a single team member to the PR. This is the person who is responsible for moving the PR forward.
+
+If a PR is still in progress and in need of early feedback, then one generally adds `WIP` to the title/description
+and self-assigns.
+
+We encourage you to assign your completed PRs to one of your reviewers in order to maintain momentum and to
+distribute responsibilities and understanding. However self-assigning is permissible when an author wants to retain
+control over merging, sequencing or follow-ups related to the PR.
 
-Assigning exactly one reviewer - no more and no less - helps ensure that there's no question about who is
-responsible. Since assignees are not explicitly expected to review, it's perfectly fine to both assign and request
-a review from the same person. Once someone has been assigned to a PR, they should make a good faith effort to take
-one of the following actions within one business day:
+Assigning exactly one reviewer — no more and no less — helps ensure that there's no question about who is
+responsible. Once someone has been assigned to a PR, they should make a good faith effort to take one of the
+following actions within one business day:
```

Comment:
Minor, but "make a good faith effort" could maybe just be "try." (Good faith is assumed I hope!)